_sass/mentors.sass

```diff
@@ -0,0 +1,106 @@
+// --------------------------------
+// Mentors / Gallery / Attendees
+// --------------------------------
+
+.svgs
+  height: 0
+  width: 0
+  display: block
+  overflow: hidden
+
+.list-mentors, .list-photos, .list-attendees
+  width: 100%
+  z-index: 900
+  transform: translate3d( -5%, 0, 0 )
+  margin: 50px 0 0 0
+  position: relative
+
+  &:hover
+
+    .image-wrapper
+      opacity: 0.25
+
+      &:hover
+        opacity: 1
+        transform: scale(1.5)
+        z-index: 9999
+
+  .row
+    @extend .cf
+    display: block
+    clear: both
+
+  .row:nth-of-type(even)
+    @extend .cf
+    display: block
+    clear: both
+    transform: translate3d( 10%, 0, 0 )
```

Comment:
I think `.list-attendees .row:nth-of-type(even)` should be `transform: translate3d(7%,0,0);` 10% is too much for the little attendees images.